### ISSUE TITLE
Crash on selecting precision mode

### ISSUE THREAD

**Issue by IamStig** [Direwolf20-MC/MiningGadgets#113]
Mining Gadget 1.5.1 for 1.16.2 Only upgrade installed first distance. 
Changed to precision mode and crashed.
Forge version 33.0.61
Crash report here - https://pastebin.com/wrms889H

**Comment by MichaelHillcox** [comment]
Huh, odd, I'll take a look thanks :+1:

**Comment by IamStig** [comment]
FYI I restarted and it all worked fine and I changed out of precision mode without any problem. I also reapplied precision without crashing, I had changed to a key bind (not shift r/click)
Also an issue with Building Gadget, not being able to select a block, it is a conflict with a mod called SaveTools. I removed it and the Gadget works fine. Likely due to the Building Gadget's tag. If you think it is worth opening an issue on the BG tracker let me know
